operations/restrictions.md

```diff
@@ -468,7 +468,106 @@ This reduces the overall number of restrictions and improves system performance.
 | `MinPrice` | [Currency value](accountingitems.md#currency-value)| optional | Value of the minimum price per time unit. |
 | `MaxPrice` | [Currency value](accountingitems.md#currency-value)| optional | Value of the maximum price per time unit. |
 
+## Clear restrictions
+
+> ### Restricted!
+> This operation is currently in beta-test and as such it is subject to change. Use of this operation must be enabled per enterprise. Please contact the Technical Partner Support team in order to enable it.
+
+Deletes restrictions that [match the conditions](#matching-conditions) using the [splicing algorithm](#splicing-algorithm).
+
+This operation is intended to be used in conjunction with [Set restrictions](#set-restrictions).
+
+### Matching conditions
+
+The [spliced](#splicing-algorithm) restriction must match the specified conditions exactly. Consider the following example:
+
+A Bookable Service has restrictions A and B. Restriction A applies to resource category C1 and rate R1. Restriction B applies to resource category C1 to all rates.
+
+If the clear data specified have specified resource category C1 but do not specify any rate, only restriction B is spliced. Restriction A remains untouched, even though it is applied to resource category C1 as well.
+
+### Splicing algorithm
+
```

Comment:
Replace this section with this:

The time interval does not need to correspond to an existing restriction in the system, instead the API uses a splicing algorithm to work out how to divide up any existing restrictions to meet the specified time interval. For example:

An existing restriction in the system applies from 5th January to 25th January. As usual, time intervals are inclusive, meaning that the time interval includes both the 5th January and the 25th January.

If the [Clear restrictions](#clear-restrictions) operation is called, specifying a restriction time interval of 10th January to 20th January, i.e. within the original restriction A, then the time interval of restriction A is split into three separate intervals.

The original restriction A is deleted, and in its place new restriction B is created for the period of time from 5th January to 9th January inclusive, and new restriction C is created for the period of time from 21st January to 25th January. Thus the period 10th January to 20th January has been cleared, but without affecting other time periods.